src/connectors/components/WalletListConnector.js

```diff
@@ -0,0 +1,21 @@
+// @flow
+
+import { connect } from 'react-redux'
+
+import type { connectorProps } from '../../components/modals/WalletListModal'
+import { WalletListModal } from '../../components/modals/WalletListModal'
+import { getActiveWalletIds } from '../../modules/UI/selectors.js'
+import type { State } from '../../types/reduxTypes.js'
+
+export const mapStateToProps = (state: State): connectorProps => {
+  const wallets = state.ui.wallets.byId
+  const activeWalletIds = getActiveWalletIds(state).filter(id => !(wallets[id] != null && wallets[id].type === 'wallet:fio'))
+  return {
+    state,
```

Comment:
It looks like you no longer need the state. Please remove

Reply:
State is needed by the WalletListModal itself

Reply:
WalletListModal does not appear to use state. From what i see it passes state into CryptoExchangeWalletListRow which is used to call the selector calculateWalletFiatBalance.  Selectors should not be called from within components. Instead you should make CryptoExchangeWalletListRow a connected component and use its connector to call calculateWalletFiatBalance and pass in the result.

Reply:
Fixed, updated PR